src/main/java/com/icthh/xm/ms/entity/security/access/DynamicPermissionCheckService.java

```diff
@@ -45,10 +50,7 @@
 @Service
 public class DynamicPermissionCheckService {
 
-    /**
-     * Permission aggregation section in custom privileges
-     */
-    public static final String CONFIG_SECTION = "entity-functions";
+    protected static Set<String> PERMISSION_SECTIONS = Set.of(FunctionCustomPrivilegesExtractor.SECTION_NAME, EntityStateCustomPrivilegesExtractor.SECTION_NAME);
```

Comment:
suggest to follow open-close principle and refactor this class

we need to have ability to add new filtration and checks without class modification